src/main/riddl/Test/messages.riddl

```diff
@@ -1,40 +1,116 @@
-
-type TenantInfo is {
-  name: String,
-  primaryContact: Contact,
-  address: Address
-} briefly "Information about the tenant" described by {
-| Basic information about the tenant in order to create it with
-| no members and no organizations
-}
-
 command CreateTenants is {
-  tenantInfos: TestGateway.TenantInfo+
+  tenantInfos: TenantContext.Info+
 } briefly "A command to create multiple tenants at once" explained by {
-| This command will create one tenant for each of the TenantInfo values
-| provided. The TenantInfo has only basic contact information. Members and
-| organizations can be added in batches by another command.
+| This command will create one tenant for each of the Info values
+| provided. The Info has only basic contact information. Organizations can be added in batches by another command.
```

Comment:
Given other changes I would drop the last sentence - "Organizations can be added in batches by another command."